### PR TITLE
feat(app): daily app build with maven wallet-sdk.

### PR DESCRIPTION
**Pull request by vkubiv**

### CODE REVIEW

File: demo/app/android/app/build.gradle [hunk 1]
```diff
@@ -22,6 +22,7 @@ if (flutterVersionName == null) {
 }
 
 def walletSdkPkgUsr = localProperties.getProperty('wallet-sdk-pkg.usr') ?: System.getenv('WALLET_SDK_USR')
+def walletSdkPkgVer = localProperties.getProperty('wallet-sdk-pkg.ver') ?: System.getenv('WALLET_SDK_VER')
```
Comment (DRK3) [inline demo/app/android/app/build.gradle:25]: Is this supposed to be in here twice? (line 24 and 25)

Comment (DRK3) [reply demo/app/android/app/build.gradle:25]: Nevermind - just realized it's `USR` and `VER`

**Review by DRK3 — APPROVED** [review]

File: .github/workflows/daily.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
+#
+# Copyright Avast Software. All Rights Reserved.
```
Comment (birtony) [inline .github/workflows/daily.yml:2]: I think this should be
```suggestion
# Copyright Gen Digital Inc. All Rights Reserved.
```

File: .github/workflows/daily.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
+#
```
Comment (birtony) [inline .github/workflows/daily.yml:1]: `daily.yml` sounds a bit confusing to me

File: .github/workflows/daily.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+# Copyright Avast Software. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: "wallet-sdk ci"
```
Comment (birtony) [inline .github/workflows/daily.yml:7]: Maybe this could be `Wallet-SDK Daily Build` or something like that?

**Review by birtony — COMMENTED** [review]
Added a couple of minor comments on naming conventions, which shouldn't be a blocker. Although, we should probably have the copyright text updated before this gets merged. Looks good otherwise!

File: .github/workflows/daily.yml (hunk 3 above)
Comment (vkubiv) [reply .github/workflows/daily.yml:1]: I didn't come up with better name

**Review by birtony — APPROVED** [review]

File: .github/workflows/daily.yml (hunk 3 above)
Comment (DRK3) [reply .github/workflows/daily.yml:1]: @birtony Do you have any suggestions? Were you thinking of something more descriptive, perhaps `dailybuild.yml`?

Comment (birtony) [reply .github/workflows/daily.yml:1]: Something like that, @DRK3

**Pull request merged**